packages/http-client-csharp/emitter/src/lib/client-model-builder.ts

```diff
@@ -27,7 +27,7 @@ import { navigateModels } from "./model.js";
 import { fromSdkServiceMethod, getParameterDefaultValue } from "./operation-converter.js";
 import { processServiceAuthentication } from "./service-authentication.js";
 
-export function createModel(sdkContext: SdkContext<NetEmitterOptions>): CodeModel {
+export function createModel(sdkContext: SdkContext<NetEmitterOptions>, logger: Logger): CodeModel {
```

Comment:
nit but if you are going to be passing both sdkContext and logger around I would probably want to make a new custom

```
interface CSharpEmitterContext {
  sdkContext: SdkContext<NetEmitterOptions>;
  logger: Logger;
}


interface CSharpEmitterContext extends SdkContext<NetEmitterOptions>;{
  logger: Logger;
}
```
woudl also make it a little shorter for the type (without generic)